samcli/local/services/base_local_service.py

```diff
@@ -81,7 +82,7 @@ def service_response(body, headers, status_code):
 
 class LambdaOutputParser:
     @staticmethod
-    def get_lambda_output(stdout_stream):
+    def get_lambda_output(stdout_stream: io.BytesIO) -> Tuple[str, bool]:
```

Comment:
I wonder if `NamedTuple` or a Dataclass would provide better readability and typing. I would not block on this but just stood out to me with the update.

Reply:
I do agree it would be better, but I didn't want to introduce more changes to this PR, I've only added some typing into the functions that is been changed.